src/AltinnCore/Common/Services/Implementation/InstanceStudioSI.cs

```diff
@@ -74,11 +74,11 @@ public async Task<Instance> InstantiateInstance(StartServiceModel startServiceMo
                 Id = instanceId.ToString(),
                 InstanceOwnerId = instanceOwnerId.ToString(),
                 ApplicationId = applicationId,
-                CreatedBy = instanceOwnerId,
+                CreatedBy = instanceOwnerId.ToString(),
                 CreatedDateTime = DateTime.UtcNow,
                 CurrentWorkflowStep = currentState.State.ToString(),
                 LastChangedDateTime = DateTime.UtcNow,
-                LastChangedBy = instanceOwnerId,
+                LastChangedBy = instanceOwnerId.ToString(),
```

Comment:
This should be userId and not instanceOwnerId.

Reply:
Ok, In studio mode, the logged in user and the instanceowner are the same